src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -179,8 +179,10 @@ def load_arguments(self, _):
                    help='Role of the service principal.')
         c.argument('skip_assignment', arg_type=get_three_state_flag(),
                    deprecate_info=c.deprecate(target='--skip-assignment', hide=True), help='No-op.')
-        c.argument('show_auth_for_sdk', options_list='--sdk-auth', deprecate_info=c.deprecate(target='--sdk-auth'),
-                   help='output result in compatible with Azure SDK auth file', arg_type=get_three_state_flag())
+        c.argument('show_auth_in_json', options_list=['--sdk-auth', '--json-auth'],
```

Comment:
As long as `options_list` doesn't introduce a breaking change, internal name change is not customer-facing and shouldn't count as breaking change.

Reply:
@jiasli agree and tool fixing is in the way :D